reference/components/document-manager/creating_documents.rst

```diff
@@ -0,0 +1,93 @@
+Creating Documents
+==================
+
+The Sulu Document Manager uses interfaces to apply mapping or behaviors to
+documents (i.e. objects). These interfaces are known as *behaviors*. Behaviors
+acts upon documents.
```

Comment:
The Sulu Document Manager uses interfaces to determine how a document is handled.